docs/overview/getting-started.md

```diff
@@ -28,5 +28,5 @@ After creating the project, you can click on the arrow to expand the view. There
 Project within Blockfrost represents API access to a specific [network](/docs/start-building#available-networks). Each project is associated with a secret token called `project_id`, which you have to use in order to access the API.
 
 :::warning
-Always keep your secrets secret! `project_id` is a secret, so make sure you don't commit it to a public repository or include it in your code. When possible, use environment variables and avoid exposing `project_id`. Otherwise, someone else could piggyback on your token.
+Always keep your secrets secret! `project_id` is a secret, so make sure you don't commit it to a public repository or include it in your code. When possible, use environment variables and avoid exposing `project_id`. Ideally, setup your own application backend to avoid `project_id` leakage. Otherwise, someone else could piggyback on your token.
```

Comment:
```suggestion
Always keep your secrets secret! `project_id` is a secret, so make sure you don't commit it to a public repository or include it in your code. When possible, use environment variables and avoid exposing `project_id`. Ideally, setup your own application backend to securely store your `project_id` to prevent the leakage. Otherwise, someone else could piggyback on your token.
```

Reply:
jsut running your own backend doesn't mean you won't store the api key in client-side code

Reply:
(nitpick)